modules/andes-core/broker/src/main/java/org/wso2/andes/kernel/FlowControlManager.java

```diff
@@ -105,14 +105,14 @@
     private boolean shutDownTriggered;
 
     public FlowControlManager() {
-        // Read configured limits
-        globalLowLimit = (Integer) AndesConfigurationManager
+        // Read configured limits and multiply by 1048576 to convert MB into Bytes
+        globalLowLimit = 1048576 * (Integer) AndesConfigurationManager
                 .readValue(AndesConfiguration.FLOW_CONTROL_GLOBAL_LOW_LIMIT);
```

Comment:
This can be overflowed. Let us document the max possible value in broker.xml config.